### PR TITLE
Harder better faster stronger

### PR DESCRIPTION
**Pull request by JohanObrink**
Fixes on infra
Custom command on tunnel
Updated README

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,15 +24,11 @@
   },
   "dependencies": {
     "camelcase": "5.3.1",
+    "chalk": "^2.4.2",
     "ip": "1.1.5",
     "localtunnel": "2.0.0",
     "subcommander": "1.0.0"
   },
-  "devDependencies": {
```
Comment (raduachim) [inline package.json]: why? need those for semantic-release

**Comment by JohanObrink** [comment]
Sorry! Merge b0rk :(

På 14 oktober 2019 till 10:02:18, Radu Achim (notifications@github.com)
skrev:

**Comment by JohanObrink** [comment]
It didn’t work when I added more yml files. It’s a feature that should be
added.

På 14 oktober 2019 till 10:22:11, Radu Achim (notifications@github.com)
skrev:

File: package-lock.json [hunk 2]
```diff
@@ -4,6 +4,8 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+<<<<<<< HEAD
+=======
     "@babel/code-frame": {
```
Comment (raduachim) [inline package-lock.json]: we should recreate this package-lock. it's b0rked over 9000

**Pull request merged**